1/test/TodoItem.spec.js

```diff
@@ -0,0 +1,13 @@
+import {mount, shallowMount} from '@vue/test-utils'
+import todoitem from '@/components/TodoItem.vue'
+
+describe('TodoItem', () => {
+
+    it('renders props.msg when passed', () => {
+        const todo = 'new todo'
+        const wrapper = mount(todoitem, {
+            propsData: { todo }
```

Comment:
So I had a look and I changed the line to

```js
const todo = null
```

And I get the following error:
```
TypeError: Cannot read property 'completed' of null
```

So somewhere in your component you have a `todo.completed` wich returns `undefined` for a String.

I would say that this test case is a false negative because it says:

```js
it('renders props.msg when passed', () => { /* ... */ })
```

But it is not checking `todo.msg` but simply that text "Edit X" is being rendered. It's certainly not what the test says -> false negative.